src/ring_theory/finiteness.lean

```diff
@@ -627,13 +682,34 @@ begin
     exact ⟨r • P, alg_hom.map_smul _ _ _⟩ }
 end
 
+/-- If an additive monoid `M` is finitely generated then `add_monoid_algebra R M` is of finite
+type. -/
 instance ft_of_fg [comm_ring R] [h : add_monoid.fg M] : finite_type R (add_monoid_algebra R M) :=
```

Comment:
Hmm, would you please expand `ft` to `finite_type` here and below?
If we decide that `finite_type` is too long and it should just be `ft` then we should refactor all of mathlib in a separate PR.
(I don't really care which we choose, but we should be consistent.)